Assets/Scripts/Game.cs

```diff
@@ -72,6 +75,13 @@ void Start()
         // Create some things
         uiScaler = new UIScaler(uICanvas);
         config = new ConfigFile();
+        currentLang = 1; // English
+
+        if (config.data.Get("MoMConfig") != null)
```

Comment:
I was confused about D2E having multilanguage, the Steam App doesn't have it. That was why I created an specific MoM config to use multilanguage but the lang selection only appears after MoM importing, not D2E. 
FFG texts for D2E arent in all languages, only english. So I decided to, in a first approach, only implement multilang for MoM.

if (config.data.Get("MoMConfig") != null). There is no such [MoMConfig] tag in previous version of Valkyrie. I added in this resquest. So What I'm looking for is exactly that. Previous users won't have [MoMConfig] tag and I will add it.